tests/test_part_image.py

```diff
@@ -124,15 +124,29 @@ def test_partition_alignment(self, bitbake_path, bitbake_variables, latest_part_
         assert(parts_start[2] == parts_end[1])
         assert(parts_start[3] == parts_end[2])
 
+        if bitbake_variables["MENDER_DEVICE_TYPE"] == "qemux86_64":
+            # For both data and swap partitons a default offset is added.
+            # Offset value is equal to alignment.
+            assert(parts_start[4] == parts_start[3] + alignment)
+            assert(parts_start[5] == parts_end[4] + alignment)
+
         # Partitions should extend for their size rounded up to alignment.
         # No set size for Rootfs partitions, so cannot check them.
         # Boot partition.
         assert(parts_end[0] == parts_start[0] + align_up(boot_part_size, alignment))
         # Data partition.
-        assert(parts_end[3] == parts_start[3] + align_up(data_part_size, alignment))
+        if bitbake_variables["MENDER_DEVICE_TYPE"] == "qemux86_64":
+            data_part_index = 4
+        else:
+            data_part_index = 3
 
+        assert(parts_end[data_part_index] == parts_start[data_part_index] + align_up(data_part_size, alignment))
+        
         # End of the last partition can be smaller than total image size, but
-        # not by more than the calculated overhead..
+        # not by more than the calculated overhead.
+        # For QEMU x86_64 images Extended partition is created as the forth one.
```

Comment:
Would say "For *some* QEMU x86_64 images..."

Also, `s/forth/fourth/`.

Reply:
Will be done.